### PR TITLE
Remove CSS preprocessor from default

### PR DESCRIPTION
**Pull request by SylvainCorlay**
This allows the preprocessor to be configured at a template level.

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
@maartenbreddels what do you think of this?

File: nbconvert/exporters/exporter.py [hunk 1]
```diff
@@ -84,7 +84,6 @@ class Exporter(LoggingConfigurable):
                                   'nbconvert.preprocessors.ExecutePreprocessor',
                                   'nbconvert.preprocessors.coalesce_streams',
                                   'nbconvert.preprocessors.SVG2PDFPreprocessor',
-                                  'nbconvert.preprocessors.CSSHTMLHeaderPreprocessor',
```
Comment (SylvainCorlay) [inline nbconvert/exporters/exporter.py:87]: Ultimately, we probably should remove all exporter-specific preprocessors from the defaults, and move them up to
 - `@default(preprocessor)` in exporters
 - or further up to the template configuration for template exporters.

**Comment by maartenbreddels** [comment]
Yes, totally makes sense. Can you comment on the different configuration (style not specified) for the different configurations?

**Review by maartenbreddels — APPROVED** [review]

**Comment by SylvainCorlay** [comment]
> Can you comment on the different configuration (style not specified) for the different configurations?

The default is `JupyterStyle`, which is the JupyterLab Pygment style...

We don't have the CSS for it in the case of classic.

**Comment by SylvainCorlay** [comment]
@maartenbreddels this may require a 6.1.0 though.

**Comment by maartenbreddels** [comment]
I don't consider this a feature, more like a chore or fix. It doesn't break anything right, and too light to be called a feature? Possibly..

**Comment by SylvainCorlay** [comment]
> It doesn't break anything right

This removes highlighting for people who are doing a custom HTML template that does not inherit from `lab`, `classic`, or `reveal`, but for example only from `base`.

Currently, highlighting is broken in classic.

**Review by MSeal — APPROVED** [review]
I'm ok with a patch version for this change. It's a slight behavioral change but it's not a breaking change.

**Pull request merged**